lib/restaurant.rb

```diff
@@ -0,0 +1,13 @@
+class Restaurant
+
+  def menu
+    @menu = [
+      {dish: 'Kebab', price: '£5.00'},
+      {dish: 'Large Fries', price: '£1.50'},
+      {dish: 'Regular Fries', price: '£1.00'},
+      {dish: 'Pounded Yam', price: '£2.00'},
+      {dish: 'Egusi Soup', price: '£4.50'},
+      {dish: 'Jellof Rice', price: '£2.00'},
+    ]
```

Comment:
The menu prices are stored as strings, can you think of a problem this will cause with maths operations on them?